pycobertura/reporters.py

```diff
@@ -443,3 +443,44 @@ def generate(self):
                     )
 
         return template.render(**render_kwargs)
+
+
+class GitHubReporter(Reporter):
```

Comment:
Can we rename the class `GitHubReporter` to something more specific? I can foresee us supporting other Github features in the future.

```suggestion
class GitHubNoticeReporter(Reporter):
```

Would this be an appropriate class name? ☝️

Reply:
Afaik, these messages are called Annotation, while notice/error/warning stands for the logging level of that annotation.

Reply:
Thus `GitHubAnnotationReporter` would be a better name.